week0003/nhantd/python/week0003.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+
+class Accounts():
+
+    def __init__(self):
+        self.data = dict()
+
+    def addAccount(self, name, amount):
+        if name not in self.data and amount >= 0:
+            self.data[name] = amount
+            return 'True'
+        else:
+            return 'False'
+
+    def depositMoney(self, name, amount):
+        if name in self.data and amount >= 0:
+            self.data[name] = self.data[name] + amount
+            return 'True'
+        else:
+            return 'False'
+
+    def withdrawMoney(self, name, amount):
+        if name in self.data and amount >= 0 and amount <= self.data[name]:
+            self.data[name] = self.data[name] - amount
+            return 'True'
+        else:
+            return 'False'
+    def executeTransaction(self, inputTransaction):
```

Comment:
This should be an external function that accepts `accounts` & `inputTransaction`. Now the class `Accounts` has too many responsibilities.